github-profiles/kenichi-7356/main.css

```diff
@@ -0,0 +1,51 @@
+@import 'https://fonts.googleapis.com/css?family=roboto:300,400,500';
+.header {
+    padding: 60px;
+    text-align: center;
+    background: #1abc9c;
+    color: white;
+    font-size: 30px;
+  }
+body{
+    margin: 0 auto;
+    max-width: 50em;
+    font-family: "Roboto", "Helvetica", "Arial", sans-serif;
+    line-height: 1.5;
+    padding: 4em 1em;
+    color: #555;
+}
+
+h2{
+    margin-top: 1em;
+    padding-top: 1em
+}
+h1,
+h2,
+strong{
+    color: #333;
+}
+
+code,
+pre{
```

Comment:
- You don't use any `pre` or `code` tags on your `.html` file.

Reply:
Thank you! I already incorporated the changes